tests/apitest/src/Foundation/NSUrl.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
```

Comment:
Instead of creating a separate test can you try to include the existing monotouch-test file inside XM's api-test ?

It will require a few `#if` to make it compile under both projects but that would gives more value to both.